scripts/config/cspell/cSpell.json

```diff
@@ -45,9 +45,7 @@
     "</?thead>",
     "</?iframe",
     // Method use, e.g. Class.foo()
-    "([a-zA-Z_]\\w*)\\.[a-zA-Z_]\\w*",
-    // Observable labels like IIZII
-    "[XIZ]{3,6}",
```

Comment:
Turns out this was explaining the `accum` vs `accumulating` thing, somehow 🤷 Removing this fixes it

Reply:
Turns out cspell is [case insensitive by default](https://cspell.org/docs/case-sensitive/) which is why it was matching "izing" words. No idea why it matches "accumulating" though 🤔 

Either way, I think better to explicitly ignore the observable labels per page.